### PR TITLE
fix samples/dhry.d so it compiles

### PR DESCRIPTION
**Pull request by WalterBright**
https://issues.dlang.org/show_bug.cgi?id=17099

### CODE REVIEW

File: samples/dhry.d [hunk 1]
```diff
@@ -357,9 +357,9 @@ alias int  One_Thirty;
 alias int  One_Fifty;
 alias char Capital_Letter;
 alias bool Boolean;
-alias char Str_30 [StrLen];
-alias int  Arr_1_Dim [50];
-alias int  Arr_2_Dim [50] [50];
+alias char[StrLen] Str_30;
```
Comment (JackStouffer) [inline samples/dhry.d]: Why not show off the new alias syntax?

Comment (JackStouffer) [inline samples/dhry.d]: Also, aren't such pointless alias against the D style?

Comment (WalterBright) [inline samples/dhry.d]: The point is to match the original `dhry` code as closely as possible.

**Comment by andralex** [comment]
@JackStouffer if you find the time, another pass to modernize the code would be definitely welcome. We can't afford to keep @WalterBright busy with this. Thanks!

**Pull request merged**